manifests/example-dev/core-modules.yaml

```diff
@@ -94,24 +101,68 @@ parameters:
           eks_node_disk_size: 20
           eks_node_instance_types:
             - "m5.xlarge"
-      eks_version: 1.21
       eks_node_spot: False
+      eks_api_endpoint_private: False
   - name: eks-addons
     value:
       deploy_aws_lb_controller: True # We deploy it unless set to False
       deploy_external_dns: True # We deploy it unless set to False
       deploy_aws_ebs_csi: True # We deploy it unless set to False
       deploy_aws_efs_csi: True # We deploy it unless set to False
-      deploy_aws_fsx_csi: False # We deploy it unless set to False
       deploy_cluster_autoscaler: True # We deploy it unless set to False
       deploy_metrics_server: True # We deploy it unless set to False
       deploy_secretsmanager_csi: True # We deploy it unless set to False
-      deploy_external_secrets: False
       deploy_cloudwatch_container_insights_metrics: True # We deploy it unless set to False
-      deploy_cloudwatch_container_insights_logs: False
+      deploy_cloudwatch_container_insights_logs: True
       cloudwatch_container_insights_logs_retention_days: 7
-      deploy_amp: False
-      deploy_grafana_for_amp: False
+      deploy_kured: False
+      deploy_calico: False
+      deploy_nginx_controller: False
```

Comment:
This attribute should be changed to the below format. Instead of having multiple keys for every feature/plugin, we can move to a format of a single key with a dict as input and parse it inside the stack:

```yaml
deploy_nginx_controller: 
        - value: False
          nginx_additional_annotations:
            nginx.ingress.kubernetes.io/whitelist-source-range: "100.64.0.0/10,10.0.0.0/8"
```